clients/java/src/test/java/com/google/myanmartools/TransliterateTest.java

```diff
@@ -761,4 +768,69 @@ public void u2ZTests() {
     }
   }
 
+  @Test
+  /**
+   * Compatibility test for Z->U conversion using golden file.
+   */
+  public void z2UCompatibilityTest() {
+    String fileInputPath = "src/test/resources/com/google/myanmartools/mmgov_zawgyi_src.txt";
+    String goldenInputPath = "src/test/resources/com/google/myanmartools/mmgov_unicode_out.txt";
+
+    // Open input files.
+    try {
+        BufferedReader input = Files.newBufferedReader(Paths.get(fileInputPath), UTF_8);
+        BufferedReader golden = Files.newBufferedReader(Paths.get(goldenInputPath), UTF_8);
+
+        int lineNum = 0;
+        String zLine  = input.readLine();
+        while (zLine != null) {
+          String convertedLine = z2U.convert(zLine);
+          String goldenLine = golden.readLine();
+          assertWithMessage(
+              "Difference in line " + lineNum + "\n" +
+              "  expected = " + goldenLine + "\n" +
+              "  actual   = " + convertedLine + "\n").
+              that(convertedLine).isEqualTo(goldenLine);
+          zLine  = input.readLine();
+          lineNum ++;
+        }
+    }
+    catch (IOException ex) {
+      org.junit.Assert.fail("** Error in file open/read in z2UCompatibiityTest: " + ex.getMessage());
+    }
+  }
+
+    @Test
+  /**
+   * Compatibility test for U->Z conversion using golden file.
+   * Note that normalization of Zawgyi is needed for comparison.
+   */
+  public void u2ZCompatibilityTest() {
+    String fileInputPath = "src/test/resources/com/google/myanmartools/udhr_mya_unicode_src.txt";
+    String goldenInputPath = "src/test/resources/com/google/myanmartools/udhr_mya_zawgyi_out.txt";
+
+    // Open input files.
+    try {
+        BufferedReader input = Files.newBufferedReader(Paths.get(fileInputPath), UTF_8);
```

Comment:
Please use getResource.  See the detector compatibility test for an example:

```java
InputStream inputStream = ZawgyiDetector.class.getResourceAsStream("/com/google/myanmartools/compatibility.tsv");
```

You want something like

```java
InputStream fileInputStream = TransliterateU2Z.class.getResourceAsStream("/com/google/myanmartools/udhr_mya_unicode_src.txt");
```

Reply:
Done.